### DISCUSSION TITLE
How likely is this project to continue?

### DISCUSSION THREAD

**Discussion by SergioHLX** [violentmonkey/violentmonkey#2161] [General]
I went to update Chrome extensions & saw that violentmonkey was listed under one that I had to replace. 

On the [Chrome web store](https://chromewebstore.google.com/detail/violentmonkey/jinjaccalgkegednnccohejagnlnfdag) I see: "This extension may soon no longer be supported because it doesn't follow best practices for Chrome extensions."

**Comment by tophf** [comment]
See #1934
